BP/SecuritiesIssuance/IssuanceProcessTerms.rdf

```diff
@@ -146,7 +146,7 @@
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-dt-fd;RegularSchedule"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-bp-iss-trm;expression"/>
+				<owl:onProperty rdf:resource="&fibo-fnd-acc-cur;hasMonetaryAmount"/>
```

Comment:
This does not seem right - the domain is MonetaryAmount not MonetaryMeasure.

Reply:
I think you are right, but I also think the restriction is unnecessary.  There is an inherited restriction for the payment which includes an amount.  That can be set via an algorithm, on an ad hoc basis, or as a regularly scheduled set of payments.  The question here is whether or not the class itself is needed at all.  It seems redundant with its parent class - whose definition is a schedule for partial payments.  I'm inclined to delete the whole class, including the restriction.  What do you think?

Reply:
Nothing references it, FYI

Reply:
The class doesn't seem useful to me, but Ill approve the PR as it is, now you've made the change.